docs/docs/getting-started/metrics.md

```diff
@@ -21,71 +21,202 @@ The Keptn Metrics Server unifies and standardizes access to all this data.
+     ```
+
+     For more details about how to install Keptn, see the
+     [Installation Guide](../installation/index.md).
+
+### Expose Prometheus and get an existing metric
+
+Now we need to expose Prometheus
+and chose an existing metric to use for this exercise.
+
+1. Use port forwarding to expose prometheus and find a suitable metric:
+
+     ```shell
+     kubectl get svc
+     ```
+
+     This command shows multiple services.
+     Look for the Prometheus UI (the one on port `9090`)
+     which should be called `prometheus-operated` (no typo).
+
+1. Port forward to access the UI:
+
+     ```shell
+     kubectl -n default port-forward svc/prometheus-operated 9090
+     ```
+
+1. Go to `http://localhost:9090/` and, in the search box,
+   type `prometheus_` and select `pr`.
+   This produces a list of metrics that are defined
+
+1. For this exercise, we are going to use a single metric.
+   In real practice, you would be defining many metrics.
+   Choose a metric that has a non-zero value.
+   For example:
+
+     ```shell
+     prometheus_http_requests_total{code="200", container="prometheus", \
+          endpoint="http-web", handler="/-/healthy", instance="10.244.0.15:9090", \
+          job="kube-prom-stack-kube-prome-prometheus", namespace="default",\
+          pod="prometheus-kube-prom-stack-kube-prome-prometheus-0", \
+          service="kube-prom-stack-kube-prome-prometheus"}
+
+     ```
+
+     For a cleaner display, remove some attributes:
+
+     ```shell
+     prometheus_http_requests_total{code="200", container="prometheus", \
+          endpoint="http-web", handler="/-/healthy", \
+          job="kube-prom-stack-kube-prome-prometheus", namespace="default", \
+          service="kube-prom-stack-kube-prome-prometheus"}
+     ```
+
+     This is your metric and value as it is stored in Prometheus.
+
+## Configure Keptn for your metrics
+
+Now you must tell Keptn about the metrics you are using.j
+You do this by defining:
+
+- A
+  [KeptnMetricsProvider](../reference/crd-reference/metricsprovider.md)
+  resource to define the external observability platform you are using
+  as a data source.
+  For this exercise, this is the Prometheus server
+  you installed and exposed above.
+- A
+  [KeptnMetric](../reference/crd-reference/metric.md)
+  resource to define each metric query you want to pull.
+
+The steps are:
+
+1. Create a
+   [KeptnMetricsProvider](../reference/crd-reference/metricsprovider.md/)
+   resource for the observability platform you are using as a data source.
+   To do this, create a new `.yml` file with content like the following:
+
+      ```shell
+        apiVersion: metrics.keptn.sh/v1
+        kind: KeptnMetricsProvider
+        metadata:
+          name: local-prometheus
+          namespace: default
+        spec:
+          type: prometheus
+          targetServer: "http://prometheus-operated.default.svc.cluster.local:9090/"
+      ```
```

Comment:
let's move this to a separate file and use include

Reply:
Yes, this is one of the items listed in https://github.com/keptn/lifecycle-toolkit/issues/3413 to be done in a separate PR.